src/lib/bootloader/grub2.rb

```diff
@@ -188,87 +188,107 @@ def disk_order_summary
 
       locations
     end
 
     def mbr_line
-      if stage1.mbr?
+      # TRANSLATORS: summary line where %s is disk specified, can be more disks, separated by comma
+      res = if stage1.mbr?
         _(
-          "Install bootcode into MBR (<a href=\"disable_boot_mbr\">do not install</a>)"
+          "Write it into MBR of %s (<a href=\"disable_boot_mbr\">do not write</a>)"
         )
+      # TRANSLATORS: summary line where %s is disk specified, can be more disks, separated by comma
       else
         _(
-          "Do not install bootcode into MBR (<a href=\"enable_boot_mbr\">install</a>)"
+          "Do not write it into MBR of %s (<a href=\"enable_boot_mbr\">write</a>)"
         )
       end
+
+      format(res, Yast::BootStorage.boot_disks.map(&:name).join(", "))
     end
 
     def partition_line
-      if stage1.boot_partition?
+      # TRANSLATORS: summary line where %s is partition specified, can be more disks,
+      # separated by comma
+      res = if stage1.boot_partition?
         _(
-          "Install boot code into a partition with /boot " \
-            "(<a href=\"disable_boot_boot\">do not install</a>)"
+          "Write it into partition with /boot - %s " \
+            "(<a href=\"disable_boot_boot\">do not write</a>)"
         )
+      # TRANSLATORS: summary line where %s is partition specified, can be more disks,
+      # separated by comma
       else
         _(
-          "Do not install boot code into a partition with /boot " \
-            "(<a href=\"enable_boot_boot\">install</a>)"
+          "Do not write it into partition with /boot - %s " \
+            "(<a href=\"enable_boot_boot\">write</a>)"
         )
       end
+      format(res, Yast::BootStorage.boot_partitions.map(&:name).join(", "))
     end
 
     def logical_partition_line
-      if stage1.boot_partition?
+      # TRANSLATORS: summary line where %s is partition specified, can be more disks,
+      # separated by comma
+      res = if stage1.boot_partition?
         _(
-          "Install boot code into a logical partition with /boot " \
-            "(<a href=\"disable_boot_boot\">do not install</a>)"
+          "Write it into logical partition with /boot - %s " \
+            "(<a href=\"disable_boot_boot\">do not write</a>)"
         )
+      # TRANSLATORS: summary line where %s is partition specified, can be more disks,
+      # separated by comma
       else
         _(
-          "Do not install boot code into a logical partition with /boot " \
-            "(<a href=\"enable_boot_boot\">install</a>)"
+          "Do not write it into logical partition with /boot - %s " \
+            "(<a href=\"enable_boot_boot\">write</a>)"
         )
       end
+      format(res, Yast::BootStorage.boot_partitions.map(&:name).join(", "))
     end
 
     def extended_partition_line
-      if stage1.extended_boot_partition?
+      # TRANSLATORS: summary line where %s is partition specified, can be more disks,
```

Comment:
Can you please check if this comment is actually copied to the .po in the expected place?

Reply:
yes, it is there. Check is easy. just run `rake pot` and it will generate pot file that is used by translators